.github/actions/collect-info/action.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Collect information and store information'
```

Comment:
Can be shortened to "Collect and store information" - or maybe for more clarity: "Collect and store debug info"